modules/rhel-compute-updating.adoc

```diff
@@ -10,7 +10,7 @@ After you update your cluster, you must update the {op-system-base-full} compute
 
 [IMPORTANT]
 ====
-{op-system-base-full} version 7.9 and version 8.4 is supported for {op-system-base} worker (compute) machines.
+{op-system-base-full} version 8.4 and 8.5 are supported for {op-system-base} compute machines.
```

Comment:
Should "version" be "versions" if there are two?

Reply:
Yes! Good catch :+1: